### PR TITLE
New Travel Policy

### PR DESCRIPTION
**Pull request by mkoci-koca**

### CODE REVIEW

File: policies/travel-policy.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+    an individual value greater than US\$30.
+-   For expenses under US\$30 receipts do not have to be provided
+    however details of the cost should be included on the **Expenses
+    Reimbursement Form**.
+-   The Traveller should cross out items unrelated to their
+    reimbursement claim on the receipts; ensure any entertainment
+    clearly lists the attendees and relevant company details; and
+    exclude any personal expenses.
+
+The Travel Authoriser or their designated representative is responsible
```
Comment (mattcaswell) [inline policies/travel-policy.md:275]: Who is the travel authoriser? Is this different to the travel approver?

File: policies/travel-policy.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+1.  Download & Complete the **Expenses Reimbursement Form**
+2.  Ensure receipts/documentation is clearly labelled with relevant
+    details (e.g. where sharing meals, crossing out personal expenses
+    etc).
+3.  Ensure you have entered the correct bank details for your
+    reimbursement to be paid into.
+4.  Scan or photograph all applicable receipts etc and attach them to
+    the **Expenses Reimbursement Form **email.
+5.  Submit the form & scanned/photographed receipts within 30 days to
+    finance@openssl.org attention to the Travel Approver for review.
```
Comment (mattcaswell) [inline policies/travel-policy.md:292]: Here it says travel approver....but above we said travel authoriser. Does it have to be the same person that originally approved the travel, or is it different?

**Review by mattcaswell — CHANGES_REQUESTED** [review]

**Comment by paulidale** [comment]
Calling a vote on this policy as at f9f492230b08b648d6f89818987cdb42f9d34d90

We need this quickly in order to plan the upcoming face to face.  It is intended to review the policy and update in the near future.

**Comment by t-j-h** [comment]
Vote +1

**Comment by iamamoose** [comment]
+1

**Comment by paulidale** [comment]
vote: +1

File: policies/travel-policy.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+related expenses OpenSSL will reimburse, the level of reimbursement, the
+process for authorising travel and how to claim a reimbursement for
+approved travel expenses.
+
+Scope
+-----
+
+The Policy applies to all Committee Members, all persons under contract
+at OpenSSL, as well as other persons as designated by the OMC from time
+to time (subsequently known collectively as *Traveller*).
```
Comment (tom-cosgrove-arm) [inline policies/travel-policy.md:20]: (collectively probably implies plural)

```suggestion
to time (subsequently known collectively as *Travellers*).
```

Also, note that much of the document uses second-person "you", while some uses "traveller(s)", so not consistent.

**Comment by mattcaswell** [comment]
On the basis that we expect to revise this policy in the near future to take account of the various feedback comments I am happy to accept it as-is for now. It's not perfect but it is better and more complete than the previous travel policy.

Vote: +1

File: policies/travel-policy.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+
+##### Flight Class Table
+ Total Flying Time is the time from the scheduled departure of the first flight to the scheduled arrival of the last flight
+ 
+ | Total Flying Time | Flight Class | Where Flight Class is not available |
+ | --- | --- | ---|
+ | Less than 6 hours | Economy| n/a |
+ | Greater than 6 but less than 12 hours | Premium Economy | Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
+ | Greater than 12 hours | Business Class | Premium Economy or Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
+
```
Comment (mkoci-koca) [inline policies/travel-policy.md:145]: To arrive 2 days prior and lose the comfort of traveling doesn't make sense. These are 2 different things. Suppose I come 2 days before I get a benefit to settle myself from jet lag. It doesn't help if I fly for more than 6 hours. 
So I would omit this column "Where flight class not available" .

**Comment by kroeckx** [comment]
I'm voting 0

**Comment by paulidale** [comment]
The vote passes 4 to 0 with one abstain and one not voted

**Comment by paulidale** [comment]
Reopening so we don't forget about the comments

File: policies/travel-policy.md [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+Only the OMC can appoint Travel Approvers and amend the ***Travel
+Approver Table***.
+
+Travel Approver's can only authorise travel up to the limits specified
+in the ***Travel Approver Table***.
+
+  
+  | Travel Approver Table | Total Value of Travel |
+  | --- | --- |
+  | Manager and OMC member | US\$15,000 |
```
[the remark this answers is not in the document]
Comment (mkoci-koca) [reply policies/travel-policy.md:85]: I understand that it should be approved by a manager and at least one OMC member.

**Pull request closed without merging**